sbnanaobj/StandardRecord/SRCVNScore.cxx

```diff
@@ -0,0 +1,13 @@
+#include "sbnanaobj/StandardRecord/SRCVNScore.h"
+#include <limits>
+
+namespace caf
+{
+  SRCVNScore::SRCVNScore() :
+    cosmicscore(std::numeric_limits<float>::signaling_NaN()),
+    ncscore(std::numeric_limits<float>::signaling_NaN()),
+    numuscore(std::numeric_limits<float>::signaling_NaN()),
+    nuescore(std::numeric_limits<float>::signaling_NaN())
+  {
```

Comment:
Is it possible for these default values to be saved into the CAF file?

Reply:
I was copying those initial values from the existing CNNScore in cafana. If the cvn result is not present in the reco2 file, the caf ttree will be filled with nans. 
<pre>recTree->Scan("rec.slc.cvn.cosmicscore:rec.slc.cvn.nuescore")
***********************************************
*    Row   * Instance * rec.slc.c * rec.slc.c *
***********************************************
*        0 *        0 *       nan *       nan *
*        0 *        1 *       nan *       nan *
*        0 *        2 *       nan *       nan *
*        0 *        3 *       nan *       nan *
*        0 *        4 *       nan *       nan *
*        0 *        5 *       nan *       nan *
*        0 *        6 *       nan *       nan *
*        0 *        7 *       nan *       nan *
*        0 *        8 *       nan *       nan *
*        0 *        9 *       nan *       nan *
***********************************************
</pre>

Reply:
Fair enough.  I was wondering if the nans would 'play nicely' with the software or plotting tools.
Thanks @yangtj207